### ISSUE TITLE
Allow to use Nova v3

### ISSUE THREAD

**Issue by ChristianKuri** [InfinetyEs/Nova-Filemanager#125]
infinety-es/nova-filemanager 2.6 requires laravel/nova ~2.0 -> satisfiable by laravel/nova[2.0.x-dev].

Nova is now in V3

**Comment by anugupta** [comment]
As a temporary workaround, you can install it from a local repo (similar to how Nova can be installed) and change the Nova version requirement 

Steps:

- clone the repo into a folder in your Laravel  structure (`./Nova-Filemanager`)
- edit `./Nova-Filemanager/composer.json` and change the Nova version requirement from 2 to 3:
` "laravel/nova": "~3.0"`

- add the local repo in the main `composer.json` file
```
    "repositories": [
        {
            "type": "composer",
            "url": "https://nova.laravel.com"
        },
        {
            "type": "path",
            "url": "./Nova-Filemanager"
        }

    ],
```

- require the package
`composer require infinety-es/nova-filemanager:*`

**Comment by Krato** [comment]
Is now available in v2.7. Please, check it!
